src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -1626,6 +1632,26 @@ internal IDisposable CallDuringDocumentDatabaseInternalDispose(Action action)
                 return new DisposableAction(() => ActionToCallDuringDocumentDatabaseInternalDispose = null);
             }
         }
+
+        public void RaiseSubscriptionTaskAddedNotification(string subscriptionToAdd)
```

Comment:
rename param - it isn't subscription to add - I assume it was already added and that's the reason we are notified. Maybe call this subscriptionName?

Reply:
done